test/integration/integration.test.ts

```diff
@@ -453,12 +462,22 @@ describe('Integration tests', (): void => {
       })
 
       expect(outgoingPayment_.id).toBe(outgoingPayment.id)
+      expect(outgoingPayment_.receiveAmount.value).toBe(amountValueToSend)
+      expect(outgoingPayment_.sentAmount.value).toBe(amountValueToSend)
+    })
+
+    test('Get Incoming Payment', async (): Promise<void> => {
+      const incomingPayment_ = await hlb.opClient.incomingPayment.getPublic({
+        url: `${incomingPayment.id}`
+      })
+      assert(incomingPayment_.receivedAmount)
+      expect(incomingPayment_.receivedAmount.value).toBe(amountValueToSend)
```

Comment:
just a flag, for the cross currency test we'd probably want this to be `toBeGreaterThanOrEqual` due to FX rounding

Reply:
We have a p2p cross currency case which I added a similar sounding check for (expects higher/lower amount based on exchange rate): https://github.com/interledger/rafiki/pull/2608#discussion_r1543815409